velox/functions/prestosql/tests/ArithmeticTest.cpp

```diff
@@ -93,27 +95,47 @@ class ArithmeticTest : public functions::test::FunctionBaseTest {
 };
 
 TEST_F(ArithmeticTest, plus) {
-  // Test plus for intervals.
-  auto op1 = makeNullableFlatVector<int64_t>(
+  // Test plus for interval day time.
+  auto dayTimeOp1 = makeNullableFlatVector<int64_t>(
```

Comment:
Can you consider templatizing the tests as you using similar formats (except for the typecasts and min/max values) everywhere ?